firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/ShakeForFeedback.kt

```diff
@@ -0,0 +1,47 @@
+package com.googletest.firebase.appdistribution.testapp
+
+import android.app.Activity
+import android.app.Application
+import android.hardware.SensorManager
+import android.os.Bundle
+import android.util.Log
+import com.google.firebase.appdistribution.ktx.appDistribution
+import com.google.firebase.ktx.Firebase
+import com.squareup.seismic.ShakeDetector
+
+class ShakeForFeedback private constructor() : ShakeDetector.Listener,
+    Application.ActivityLifecycleCallbacks {
+    private val shakeDetector = ShakeDetector(this)
+
+    override fun hearShake() {
```

Comment:
"hear" - interesting terminology, but I guess that's what Square picked...

Reply:
Yeah true, you'd think it would be "feelShake" :)

Maybe because it's a "Listener"?

Reply:
🤯